content/javascript/concepts/objects/objects.md

```diff
@@ -96,3 +96,69 @@ This will output:
 ```shell
 Hi, I'm Elizabeth Harmon.
 ```
+
+## Object Classes
+
+Classes are essentially boiler plate object templates. If a car was an object, then a car factory is an object class.
+
+A class can be constructed with the following notation. The person object example will be used. **Note: anonymous functions can't be used in classes**
```

Comment:
- [ ] Notes should be written like so: **Note:** This is the note description.
- [ ] **Note:** Anonymous functions can't be used in classes.